### PR TITLE
Add description to Descendant tree report-styles

### PR DESCRIPTION
**Pull request by FredrikLindseth**
The descendant tree style editor was missing some descriptions so I added them

### CODE REVIEW

File: gramps/plugins/drawreport/descendtree.py [hunk 1]
```diff
@@ -1814,30 +1814,32 @@ def make_default_style(self, default_style):
         graph_style = GraphicsStyle()
         graph_style.set_paragraph_style("CG2-Normal")
         graph_style.set_fill_color((255, 255, 255))
+        para_style.set_description(_("The style for the marrige-box."))
```
Comment (fxtmtrsine) [inline gramps/plugins/drawreport/descendtree.py]: Spelling:  marrige > marriage

File: gramps/plugins/drawreport/descendtree.py [hunk 2]
```diff
@@ -1814,30 +1814,32 @@ def make_default_style(self, default_style):
...
+        para_style.set_description(
+            _("The style for the line connection persons."))
         default_style.add_draw_style("CG2-line", graph_style)
-
-#=====================================
-#So do not fear, for I am with you; do not be dismayed,
-#for I am your God.  I will strengthen you and help you;
-#I will uphold you with my righteous right hand.
-#Isaiah 41:10
+
```
Comment (fxtmtrsine) [inline gramps/plugins/drawreport/descendtree.py]: No need to remove this as it was added by the original developer of report.

**Comment by fxtmtrsine** [comment]
fails on travis due to [trailing white space](https://travis-ci.org/gramps-project/gramps/builds/552396366#L2825)

**Comment by Nick-Hall** [comment]
Thanks for your contribution.

Merged with slight alterations in commit 52e76b506aff5a5bec0dd14163a431bb49585236.

**Pull request closed without merging**